shell/gpu/gpu_surface_gl.cc

```diff
@@ -61,6 +61,9 @@ sk_sp<GrDirectContext> GPUSurfaceGL::MakeGLContext(
   // TODO(goderbauer): remove option when skbug.com/7523 is fixed.
   // A similar work-around is also used in shell/common/io_manager.cc.
   options.fDisableGpuYUVConversion = true;
+
+  options.fReduceOpsTaskSplitting = GrContextOptions::Enable::kYes;
```

Comment:
This is only in effect in the GL backend. The identical change must be applied to `gpu_surface_vulkan` and `gpu_surface_metal`.